BUILD.bazel

```diff
@@ -912,6 +912,19 @@ cc_library(
     ],
 )
 
+cc_test(
```

Comment:
Lets think more about how this will integrate into other projects that rely on StableHLO. I agree this should run in our CI, but I do not think that it should run in other projects that may manually manage their LLVM revisions.